isis/src/system/apps/isisdataeval/tools/README.md

```diff
@@ -17,29 +17,31 @@ The ISISDATA mockup system is a complete copy of every directory and every file
 
 # <a name="#isisdatamockuptool">ISISDATA Mockup Tool - isisdata_mockup.py</a>
 
-A tool has been provided with the `isisdataeval` application that generates ISISDATA mockups for this and potentially other purposes. This Python tool will convert an ISISDATA installation into a mockup that is suitable to test the database lookup system and verify the contents and structure of mission kernel configurations in $ISISDATA. However, it has been generalized to use for any directory structure. Here is the application documentation:
+A tool has been provided with the `isisdataeval` application that generates ISISDATA mockups for this and potentially other uses. This Python tool will convert an ISISDATA installation into a mockup that is suitable to test the database lookup system and **verify** the contents and structure of mission kernel configurations in $ISISDATA. However, it has been generalized to use for any directory structure. Here is the application documentation:
 
 `isisdata_mockup.py -h`
 
 ```
-usage: isisdata_mockup.py [-h] --isisdata ISISDATA --outpath OUTPATH [--ghostdir GHOSTDIR]
-                          [--saveconfig] [--dryrun] [--verbose]
+usage: isisdata_mockup.py [-h] --isisdata ISISDATA [--outpath OUTPATH] [--ghostdir GHOSTDIR]
+                          [--hasher {md5,sha1,sha256,all}] [--saveconfig] [--tojson TOJSON] [--dryrun] [--verbose]
 
 Convert ISISDATA directory to test format
 
 optional arguments:
-  -h, --help           show this help message and exit
-  --isisdata ISISDATA  ISISDATA directory root
-  --outpath OUTPATH    NEW ISISDATA directory to create
-  --ghostdir GHOSTDIR  Replaces the --isisdata path with this string to ghost the actual input
-                       directory path
-  --saveconfig, -s     Retain *.db, *.conf files rather than replace with processs info
-  --dryrun, -n         Only print actions but do not execute
-  --verbose, -v        Verbose output
+  -h, --help            show this help message and exit
+  --isisdata ISISDATA   ISISDATA directory root
+  --outpath OUTPATH     NEW ISISDATA directory to create
+  --ghostdir GHOSTDIR   Replaces the --isisdata path with this string to ghost the actual input directory path
+  --hasher {md5,sha1,sha256,all}
+                        Add desired/supported hash algorithms to compute
+  --saveconfig, -s      Retain *.db, *.conf files rather than replace with processs info
+  --tojson TOJSON       Write all ISISDATA file information to output file in JSON format
+  --dryrun, -n          Only print --outpath actions but do not execute
+  --verbose, -v         Verbose output
```

Comment:
could we just avoid putting in-the-weeds parameter details in the readme and instead instruct to run the command? this way we don't have to keep this in sync.

Reply:
Sure make sense. I removed the help and added additional context.

Reply:
Looks like the full output of the help message is still there?

Reply:
This section was removed/updated in [this push](https://github.com/DOI-USGS/ISIS3/pull/5163/commits/498c9f4836ee882efe76a9dc9570926341c5eece) on or about April 3. You can also check the [source](https://github.com/KrisBecker/ISIS3/blob/feature/isisdataeval-update/isis/src/system/apps/isisdataeval/tools/README.md) to see the changes.

Reply:
weird, idk why Github it doing me like this.